src/daft-core/src/series/ops/list.rs

```diff
@@ -175,4 +175,40 @@ impl Series {
             ))),
         }
     }
+
+    pub fn list_contains(&self, contains: &Self) -> DaftResult<Self> {
+        if contains.len() != 1 {
+            return Err(DaftError::ValueError(
+                "Contains must be a single value".to_string(),
+            ));
+        }
```

Comment:
We could technically allow multiple `contains` too, each one corresponding to the list at the same index.

Reply:
Ya, I think during implementation I had this thought as well. Will try to implement it.

Reply:
Updated.